src/Request.js

```diff
@@ -50,6 +50,20 @@ module.exports = Class.extend({
       return null;
    },
 
+   /**
+    * Returns the type and subtype, e.g. 'application/json', of the MIME type found in the
+    * Content-Type header.
+    *
+    * @see https://mimesniff.spec.whatwg.org/#mime-type-essence
+    * @return {string} The essence of the MIME or `undefined` if the Content-Type header
+    * is not found
+    */
+   _getContentTypeEssence: function() {
+      var contentType = this.header('Content-Type');
+
+      return _.isString(contentType) ? contentType.replace(/;.*/, '').trim() : undefined;
```

Comment:
Do you think we need to lower-case the returned value (i.e. add `.toLowerCase()` after `.trim()`)? This library was previously only used for service-to-service APIs, so we didn't have to deal with the crazy variety of things that can come from browser-world.